### DISCUSSION TITLE
More details?

### DISCUSSION THREAD

**Discussion by djamps** [tomaskovacik/IS2020#4] [General]
Hi Tomas, thank you for creating this.   I just ordered your eval boards.   I will need to change some settings in firmware, so I wonder if this is the right project?  I looked at the code and it seems it is about controlling (phonebook, paus/play, ect).  I am very familiar with arduino but not using them with CPP.   

Also, your blog post about firmware updates mentions STM32.   Can this be done in arduino?   

Thanks for pointing me the right way.

**Comment by tomaskovacik** [comment]
arduino which is using atmega16u/8u as a converter from USB to serial should work, but I cannot remember if it works or not, but in Linux it is actual modem device not serial port, which is probably important part for microchip software recognise device as supported port

also you can upload firmware using USB same as for bm83, https://kovo-blog.blogspot.com/2021/05/bm83-how-to-s.html

**Reply by tomaskovacik** [reply]
regarding documentation, check documents for development board https://www.microchip.com/developmenttools/ProductDetails/PartNO/BM-64-EVB-C1

**Reply by djamps** [reply]
Ah, so it is  a UART interface for firmware?   I can do that with easy in arduino.

Have you tried concert or stereo mode in these devices?   Seems a really unique feature for the eval board, but with no mention in the BM64 datasheet.

**Reply by tomaskovacik** [reply]
I am talking about CDC/ACM device https://www.keil.com/pack/doc/mw/USB/html/group__usbd__cdc_functions__acm.html 

I do not try wireless stereo, I am using all my modules as sink, not the source, but bm64 has this picture:
![image](https://user-images.githubusercontent.com/4346464/120033421-cf447a00-bffb-11eb-8401-cab4fbbb4535.png)
